driver/scsi_driver_extensions.c

```diff
@@ -113,7 +113,7 @@ WnbdHwFindAdapter(PVOID DeviceExtension,
     /*
      * https://docs.microsoft.com/en-us/previous-versions/windows/hardware/drivers/ff563901(v%3Dvs.85)
      */
```

Comment:
Nit: Can you please add a comment that although using `SP_UNINITIALIZED_VALUE` we receive zero lengths for `SCSIOP_READ|SCSIOP_WRITE`